rfcs/spinner-component.md

```diff
@@ -44,7 +44,7 @@ export default () => {
 
 * `size: number|string` - Optional, Defaults to 44
   The size of the circle. Is passed to the underlying svg width/height style. Can also be a baseui `theme.sizing` constant like `scale500`
-* `color: string` - Optional, Defaults to `theme.primary450`
+* `color: string` - Optional, Defaults to `theme.primary400`
```

Comment:
Why not just call these `theme.primary`?

Reply:
good point! will update